MANIFEST.in

```diff
@@ -2,5 +2,5 @@ include setup.py
 include CHANGES.txt
 include LICENSE.txt
 include README.rst
-include runtests.py
-recursive-include djmoney *.py *.html *.txt *.png *.js *.css *.gif *.less *.mo *.po *.otf *.svg *.woff *.eot *.ttf
+recursive-include djmoney *.py
+recursive-include tests *.py
```

Comment:
This `tests` should not be included as we are not going to install it on a user's system and having a global `tests` package would probably create conflict